site/js/localization.js

```diff
@@ -257,8 +257,8 @@ function prettyPrintDateString(date) {
     var d = new Date(date)
     let localeDate = d.toLocaleString(getLocale(), {
         weekday: "long",
-        day: "numeric",
-        year: "numeric",
+        day: "Numberic",
```

Comment:
Is "Numberic" a valid option? According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString, "numeric" seems to be correct. Am I missing something here?

Reply:
Just a stupid search and replace error ..😒